### DISCUSSION TITLE
The onboarding experience for new users is bad

### DISCUSSION THREAD

**Discussion by Poopooracoocoo** [community/community#54181] [New to GitHub]
It feels slow and like something to endure. I saw them type in the code manually as they couldn't drag and drop it it from their email. I could go on and on. It just looks like something that is meant to deter new users. It's an interesting strategy of using UX to do so.

**Comment by queenofcorgis** [comment]
Thanks for your feedback - we're always looking to improve - feel free to share more details if so inclined!
